source/jormungandr/jormungandr/instance.py

```diff
@@ -719,6 +718,16 @@ def get_street_network(self, mode, request):
             streetnetwork_backend_conf = getattr(self, column_in_db)
             return self._streetnetwork_backend_manager.get_street_network_db(self, streetnetwork_backend_conf)
 
+    def get_street_network(self, mode, request):
+        if mode != fallback_modes.FallbackModes.car.name:
+            return self._get_street_network(mode, request)
+
+        walking_service = self._get_street_network(fallback_modes.FallbackModes.car.name, request)
```

Comment:
why this is not `fallback_modes.FallbackModes.walking.name` ?

Reply:
copy paste is evil !